ui/apps/dashboard/src/components/FeatureFlags/ServerFeatureFlag.tsx

```diff
@@ -23,25 +23,34 @@ export async function getBooleanFlag(
   { defaultValue = false }: { defaultValue?: boolean } = {}
 ): Promise<boolean> {
   const user = await currentUser();
-  const client = await getLaunchDarklyClient();
-
-  const accountID =
-    user?.publicMetadata.accountID && typeof user?.publicMetadata.accountID === 'string'
-      ? user?.publicMetadata.accountID
-      : 'Unknown';
-
-  const context = {
-    account: {
-      key: accountID,
-      name: 'Unknown', // TODO: Add account name whenever we have adopted Clerk Organizations
-    },
-    kind: 'multi',
-    user: {
-      anonymous: false,
-      key: user?.externalId ?? 'Unknown',
-      name: `${user?.firstName ?? ''} ${user?.lastName ?? ''}`.trim() || 'Unknown',
-    },
-  } as const;
-
-  return client.variation(flag, context, defaultValue);
+
+  try {
+    const client = await getLaunchDarklyClient();
+
+    const accountID =
+      user?.publicMetadata.accountID && typeof user?.publicMetadata.accountID === 'string'
+        ? user?.publicMetadata.accountID
+        : 'Unknown';
+
+    const context = {
+      account: {
+        key: accountID,
+        name: 'Unknown', // TODO: Add account name whenever we have adopted Clerk Organizations
+      },
+      kind: 'multi',
+      user: {
+        anonymous: false,
+        key: user?.externalId ?? 'Unknown',
+        name: `${user?.firstName ?? ''} ${user?.lastName ?? ''}`.trim() || 'Unknown',
+      },
+    } as const;
+
+    const variation = await client.variation(flag, context, defaultValue);
+    return variation;
+  } catch (err) {
+    console.error('Failed to get LaunchDarkly variation', err);
```

Comment:
could you send the error to Sentry so we can see it as well.
otherwise, not sure which blackhole this error will disappear to.